dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/filter/GenericFilter.java

```diff
@@ -162,7 +166,9 @@ public Result invoke(Invoker<?> invoker, Invocation inv) throws RpcException {
                     }
                 }
 
-                RpcInvocation rpcInvocation = new RpcInvocation(method, invoker.getInterface().getName(), invoker.getUrl().getProtocolServiceKey(), args, inv.getObjectAttachments(), inv.getAttributes());
+                RpcInvocation rpcInvocation =
```

Comment:
why format this line？

Reply:
This code is too long, in order to facilitate reading, so formatted.